src/main/java/ai/deepcode/javaclient/DeepCodeRestApi.java

```diff
@@ -363,13 +363,14 @@ public static EmptyResponse UploadFiles(
   }
 
   private interface GetAnalysisCall {
-    //    @retrofit2.http.Headers("Content-Type: application/json")
-    @GET("analysis/{bundleId}")
+    @retrofit2.http.Headers("Content-Type: application/json")
+    @POST("analysis/{bundleId}")
     Call<GetAnalysisResponse> doGetAnalysis(
```

Comment:
it's POST request now: maybe changing method name for readability to `doPostAnalysis`?

Reply:
It's the wrapper for "Get Analysis" api call.
Better to leave it as is to easily match